service/src/main/java/aero/minova/cas/service/SecurityService.java

```diff
@@ -62,33 +67,103 @@ public boolean isTablePresent(String tableName) throws Exception {
 
 	/**
 	 * Überprüft, ob es in der vCASUserPrivileges mindestens einen Eintrag für die User Group des momentan eingeloggten Users gibt. Die Abfrage sieht
-	 * folgendermaßen aus: select PrivilegeKeyText,KeyText,RowLevelSecurity from xvcasUserPrivileges where (PrivilegeKeyText = privilegeName and KeyText =
-	 * UserSecurityToken1) or (PrivilegeKeyText = privilegeName and KeyText = UserSecurityToken2) or ... Die erzeugten Rows haben folgendes Format: Row r =
-	 * [Tabellenname,UserSecurityToken,RowLevelSecurity], Beispiel: Row r = ["tTestTabelle","User1",1], Row r2 = ["tTestTabelle","User2",0]
+	 * folgendermaßen aus: select PrivilegeKeyText,KeyText,RowLevelSecurity from xvcasUserSecurity where (PrivilegeKeyText = privilegeName and SecurityToken =
+	 * UserSecurityToken1) or (PrivilegeKeyText = privilegeName and SecurityToken = UserSecurityToken2) or ... Die erzeugten Rows haben folgendes Format: Row r
+	 * = [Tabellenname,UserSecurityToken,RowLevelSecurity], Beispiel: Row r = ["tTestTabelle","User1",1], Row r2 = ["tTestTabelle","User2",0]
 	 *
 	 * @param privilegeName
 	 *            Das Privilege, für das ein Recht eingefordert wird.
 	 * @return Enthält alle Gruppen, die Ein Recht auf das Privileg haben.
 	 **/
 	public List<Row> getPrivilegePermissions(String privilegeName) {
+		loadAllPrivileges();
 		@SuppressWarnings("unchecked")
 		List<GrantedAuthority> allUserAuthorities = (List<GrantedAuthority>) SecurityContextHolder.getContext().getAuthentication().getAuthorities();
 		Table userPrivileges = new Table();
-		userPrivileges.setName("xvcasUserPrivileges");
+		userPrivileges.setName("xvcasUserSecurity");
 		List<Column> columns = new ArrayList<>();
 		columns.add(new Column("PrivilegeKeyText", DataType.STRING));
-		columns.add(new Column("KeyText", DataType.STRING));
+		columns.add(new Column("SecurityToken", DataType.STRING));
 		columns.add(new Column("RowLevelSecurity", DataType.BOOLEAN));
 		columns.add(Column.AND_FIELD);
 		userPrivileges.setColumns(columns);
 
 		for (GrantedAuthority ga : allUserAuthorities) {
+
+			// Überprüfen, ob der SecurityToken an irgendeiner Position in der SecurityToken-Spalte auftaucht.
 			Row tableNameAndUserToken = new Row();
+
+			// Token ist in der Mitte des Strings.
+			tableNameAndUserToken.setValues(
+					asList(new Value(privilegeName, null), new Value("%#" + ga.getAuthority() + "#%", null), new Value("", null), new Value(false, null)));
+			// Token ist am Ende des Strings
 			tableNameAndUserToken
-					.setValues(asList(new Value(privilegeName, null), new Value(ga.getAuthority(), null), new Value("", null), new Value(false, null)));
+					.setValues(asList(new Value(privilegeName, null), new Value("%#" + ga.getAuthority(), null), new Value("", null), new Value(false, null)));
```

Comment:
Bräuchte man hier nicht mehrere likes?
Also nicht nur `%#[...]` und `%#[...]#%`, sondern auch `[...]#%`?
Müsste das zudem nicht in den 2ten Parameter von Value rein, weil es eine Regel ist?

Reply:
Der Fall, dass der Token am Anfang ist, ist bereits durch `%#[...]`.